.meta/transforms/remove_fields.toml

```diff
@@ -17,3 +17,10 @@ examples = [["field1", "field2", "parent.child"]]
 field_path_notation = true
 required = true
 description = "The log field names to drop."
+
+[transforms.remove_fields.options.drop_empty]
+type = "bool"
+common = true
```

Comment:
I'd remove this. I don't think this is a common option and will likely confuse users. Curious if you agree?

Reply:
If you think so, I will remove. I set it `true` under the assumption that "common" options were shared across multiple units (in common) as opposed to commonly used.

Reply:
Ah.  I meant for it to be commonly used options, to reduce the noise for the initial configuration examples.